test/test_prototype_transforms.py

```diff
@@ -269,6 +270,20 @@ def test_common(self, transform, adapter, container_type, image_or_video, device
             else:
                 assert output_item is input_item
 
+        sanitize = transforms.SanitizeBoundingBoxes()
+        for output_degenerate_bounding_box, input_degenerate_bounding_box in (
+            (output_item, input_item)
+            for key, output_item, input_item in zip(keys, output_flat, input_flat)
+            if "degenerate" in key
+        ):
+            sample = dict(
+                bounding_box=output_degenerate_bounding_box,
+                labels=torch.randint(
+                    10, input_degenerate_bounding_box.shape[:-1], device=input_degenerate_bounding_box.device
+                ),
+            )
+            assert sanitize(sample)["bounding_box"].shape == (0, 4)
```

Comment:
Unfortunately, this fails for the following transforms:

- `ElasticTransform`
- `RandomAffine`
- `RandomPerspective`
- `RandomRotation`

So all of our affine transformations.

Maybe the test is a little too strict and we should only check what happens to `[0, 0, 0, 0]` since that is what `RandomIoUCrop` sets to?

Reply:
> Maybe the test is a little too strict and we should only check what happens to [0, 0, 0, 0] since that is what RandomIoUCrop sets to?

Let's go for that right now, with a TODO prompting to investigate the other degenerate cases?